src/Rector/Closure/ServiceSettersToSettersAutodiscoveryRector.php

```diff
@@ -106,9 +105,6 @@ public function refactor(Node $node): ?Node
         );
 
         $sharedNamespace = $this->minimalSharedStringSolver->solve(...$classNames);
-        if (! is_string($sharedNamespace)) {
-            return null;
-        }
```

Comment:
since the `MinimalSharedStringSolver` no longer implements the `Solver` interface, we were able to declare more specific return types which also means this case - which was never taken, as `null` could technically not be returned - can be dropped.